src/content/app/genome-browser/components/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -15,163 +15,42 @@
  */
 
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { OutgoingActionType } from '@ensembl/ensembl-genome-browser';
+import { useSelector } from 'react-redux';
+
+import RadioGroup, {
+  RadioOptions
+} from 'src/shared/components/radio-group/RadioGroup';
```

Comment:
Just curious which one is better: to keep this `this track/all tracks` toggle in this component, or to copy it across the individual track config panel components?

Reply:
I am not sure which one is good. It is a shared component across all types of track configs. Will this change in the future? May be we can ask Andrea?